Formula/uftrace.rb

```diff
@@ -0,0 +1,29 @@
+class Uftrace < Formula
+  desc "Function graph tracer for C/C++/Rust"
+  homepage "https://uftrace.github.io/slide/"
+  url "https://github.com/namhyung/uftrace/archive/refs/tags/v0.10.tar.gz"
+  sha256 "b8b56d540ea95c3eafe56440d6a998e0a140d53ca2584916b6ca82702795bbd9"
+  license "GPL-2.0-only"
+  head "https://github.com/namhyung/uftrace.git", branch: "master"
+
+  depends_on "pandoc" => :build
+  depends_on "capstone"
+  depends_on "elfutils"
+  depends_on "libunwind"
+  depends_on :linux
+  depends_on "ncurses"
+  depends_on "python@3.9"
+
+  def install
+    system "./configure", *std_configure_args, "--ldflags=-lcapstone"
```

Comment:
Probably should try adding `pkg-config` as build dependency and see if it helps pick up the `ldflags`/dependencies. From previous failure logs, configure didn't find `ncurses`, 
```
...    libncursesw: [ OFF ] - TUI support
```

https://github.com/namhyung/uftrace/blob/master/INSTALL.md#dependency
> Firstly, please make sure `pkg-config` is installed in the system to properly detect the dependencies of uftrace. Otherwise, some packages may not be detected even if they are already installed and it disables some features of uftrace.

Reply:
Initially, I didn't use 'depends_on "ncurses"', yet. After adding it, I saw the executable indicated it has TUI support.

But adding pkg-config seems to fix the missing -lcapstone. Pushed with 'depends_on "pkg-config" => :build' now.